### ISSUE TITLE
Support java.io.InputStream

### ISSUE THREAD

**Issue by markthagan** [OWASP/json-sanitizer#18]
It is a bit limiting to force the JSON to be passed as a String especially for use in cases where the message size/volume may be large. The JAXRS MessageBodyReader API provides an InputStream. Is there a plan to extend the API to accept and return an InputStream so that we can maintain optimised access to the data as a stream?

**Comment by jmanico** [comment]
We don’t have a plan but we accept PR’s if you’re into it!

--
Jim Manico
@Manicode

**Comment by markthagan** [comment]
Apologies if this is a duplicate response as I tried to reply in email.

What is a PR?

**Comment by markthagan** [comment]
Never mind, sorry. RTFM. Must be Pull Request as-in, develop it myself and if its any good we can share?

**Comment by jmanico** [comment]
Pull request?

--
Jim Manico
@Manicode
Secure Coding Education
+1 (808) 652-3805

**Comment by jmanico** [comment]
Exactly! :) if you submit a change we’ll work with you to integrate it into the project.

--
Jim Manico
@Manicode
